<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetLogin</class>
 <widget class="QWidget" name="WidgetLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>273</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>331</width>
    <height>273</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>331</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidgetLogin">
     <property name="minimumSize">
      <size>
       <width>331</width>
       <height>273</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="TabPhoneLogin">
      <attribute name="title">
       <string>PhoneLogin</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="horizontalSpacing">
              <number>10</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="labelCountry">
               <property name="text">
                <string>Country</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <property name="spacing">
                <number>1</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="labelPLoginInfo">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_13">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="labelPhonePassword">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>72</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QPushButton" name="pBLostPhonePassword">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string>Forget</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0" colspan="3">
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="labelPPassWordInfo">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_10">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="lEPhonePassword">
               <property name="minimumSize">
                <size>
                 <width>110</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maxLength">
                <number>16</number>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="labelPhoneUserName">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>72</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string>Phone</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QPushButton" name="pBRegisterByPhone">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string>Register</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,7">
               <property name="spacing">
                <number>1</number>
               </property>
               <item>
                <widget class="QLineEdit" name="lEISO">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maxLength">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="CPhoneNumRegExp" name="lEPhoneUserName">
                 <property name="minimumSize">
                  <size>
                   <width>70</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <kerning>true</kerning>
                  </font>
                 </property>
                 <property name="maxLength">
                  <number>32</number>
                 </property>
                 <property name="frame">
                  <bool>true</bool>
                 </property>
                 <property name="cursorMoveStyle">
                  <enum>Qt::LogicalMoveStyle</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="8" column="1">
              <widget class="QPushButton" name="pBPhoneLogin">
               <property name="minimumSize">
                <size>
                 <width>110</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string>Login</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="3">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="labelPUserInfo">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="MyComboBox" name="cbxCountryCode">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>32</height>
                </size>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
              </widget>
             </item>
             <item row="6" column="1" colspan="2">
              <widget class="QCheckBox" name="cBSavePPassword">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string>Remember Password</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabMailLogin">
      <attribute name="title">
       <string>MailLogin</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0">
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="5" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="labelMLoginInfo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_15">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="0" column="0" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="labelMUserInfo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lEMailUserName">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <property name="maxLength">
              <number>32</number>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="pBLostMailPassWord">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Forget</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelMailUserName">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>72</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Mail</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelMailPassword">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>72</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Password</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lEMailPassword">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pBRegisterByMail">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Register</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="labelMPasswordInfo">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_12">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="pBMaiLogin">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" colspan="2">
            <widget class="QCheckBox" name="cBSaveMPassword">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Remember Password</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPhoneNumRegExp</class>
   <extends>QLineEdit</extends>
   <header>../zxLib/widget/phonenumregexp.h</header>
  </customwidget>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>../Common/widget/mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidgetLogin</tabstop>
  <tabstop>cbxCountryCode</tabstop>
  <tabstop>lEISO</tabstop>
  <tabstop>lEPhoneUserName</tabstop>
  <tabstop>lEPhonePassword</tabstop>
  <tabstop>cBSavePPassword</tabstop>
  <tabstop>pBPhoneLogin</tabstop>
  <tabstop>pBRegisterByPhone</tabstop>
  <tabstop>pBLostPhonePassword</tabstop>
  <tabstop>lEMailUserName</tabstop>
  <tabstop>lEMailPassword</tabstop>
  <tabstop>cBSaveMPassword</tabstop>
  <tabstop>pBMaiLogin</tabstop>
  <tabstop>pBRegisterByMail</tabstop>
  <tabstop>pBLostMailPassWord</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
